public_suffix_list.dat

```diff
@@ -13231,6 +13231,10 @@ myshopblocks.com
 // Submitted by Alex Richter <alex.richter@shopify.com>
 myshopify.com
 
+// Hoplix : https://www.hoplix.com
```

Comment:
Please sort this according to the Guidelines - https://github.com/publicsuffix/list/wiki/Guidelines

Reply:
Please also provide a more substantial rationale for the request

Reply:
Ok i sorted the lines and also i provided more substantial retionale for the request.
Thanks for advice